### ISSUE TITLE
tarball and tag for the latest release differ

### ISSUE THREAD

**Issue by agaida** [andrew-bibb/cmst#125]
- not the best idea to be true - one should tag a release in git once  it's really done
- please don't include dummy-directories with placeholder files - qmake don't need it
- please don't include Makefiles in the tarball - qmake will create such things
- please don't include binaries in the tarball

LANG=C siduction-paste  diff -r -u tarball/ cmst-2016.01.26: http://paste.siduction.org/20160128225115 

cheers Alf

**Comment by andrew-bibb** [comment]
Actually it was done when I uploaded it, but I had a typo in the name (16 instead of 26).  Thought I had fixed that, but still see a file there with the 16 in it.  I may just delete the entrire release and try again.

Dummy directories I've been having problems with which is why I put the placeholder files in.  I'd love to remove them (they are a pain to keep straight) so let me try again

Agree, there should be no make files in the tarball. Same with the binaries.  Now I'm wondering what I did.  I think nuke this one and start again.  Back in a bit.

**Comment by agaida** [comment]
thank you very much - i repacked the tar ball and package it - compiles and run without problems, Great Work.

**Comment by andrew-bibb** [comment]
Oh man, I must have been brain dead when I uploaded that last release.  Anyway it was so bad that I did just delete it and start again with another. 

Also managed to remove those dummy directories.  Not sure now why I put them in at all (didn't use to have to - that was a more recent addition), but they are removed and it builds fine, so good riddance.

As I was typing this I saw your last message, so I'm going to close this out.  Thank you for pointing this out.

A-
